syft/frameworks/types.py

```diff
@@ -0,0 +1,38 @@
+# from __future__ import absolute_import
```

Comment:
Can we git rid of this comment. We don't support python 2 (or any python below 3.6)

Reply:
sure thing, nice catch